examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,57 @@
+"""
+Using distributed for single_machine parallel computing
```

Comment:
single_machine -> single machine

Reply:
FWIW we're trying to avoid referring the to code in the github.com/dask/distributed repository as `distributed`. The reason here is that it's a fairly generic term.  Instead I might recommend just using the term `dask` here, or, if preferred, `dask.distributed`.

Reply:
I'll use dask.distributed.

This naming thing is quite confusing (I can't blame you, it's an error that I been into over and over, first with enthought.mayavi => mayavi, and later with scikits.learn => scikit-learn (imported as sklearn)). However, it will confuse the users, and even myself, as it makes the difference and the boundary between projects quite blurry.